getstarted.md

```diff
@@ -120,28 +110,20 @@ The final step adds `react-hot-loader` to our project to preserve *component sta
     a.  If you use Babel, modify your `.babelrc` to ensure it includes at least:
 
     ```js
-    {
-      "plugins": [ "react-hot-loader/babel" ]
-    }
+      {
+        "plugins": [ "react-hot-loader/babel" ]
+      }
     ```
-    b. Alternatively, in Webpack, add `react-hot-loader/webpack` to your loaders
+    b. Alternatively, in Webpack, add `react-hot-loader/webpack` to the **loaders** section of your `webpackConfig.js`:
 
     ```js
-        // webpackConfig.js
-
-        // TODO: Would love some help showing the shape of the webpack config without
-        // overwhelming users either - just want it to be familiar enough. I suppose we could
-        // also declare a variable and assign the require statement to it? (Just an idea)
-        devtool: ...,
-        entry: [...],
         module: {
             loaders: [{
                 test: /\.js$/,
                 loaders: ['react-hot-loader/webpack', 'babel'],
                 include: path.join(__dirname, 'src')
             }]
         }
-
     ```
 
     NB: `react-hot-loader/webpack` only works on *exported* components,
```

Comment:
Oops, almost missed this note.  Yeah, it came up already in the main thread and is definitely important for people to understand.  We can make it clearer and add to the troubleshooting docs too.